modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -867,6 +871,13 @@ private void _addObjectDefinitionResources(
 							objectAction.getExternalReferenceCode(),
 							objectDefinitionId);
 
+				Map<String, String> labelMap = new HashMap<>(
+					objectAction.getLabel());
+
+				labelMap.putIfAbsent(defaultLanguageId, objectAction.getName());
```

Comment:
I would change this part to this:
```
        String siteDefaultLanguageId = LocaleUtil.toLanguageId(LocaleUtil.getSiteDefault());

	Map<String, String> labelMap = new HashMap<>(objectAction.getLabel());

	labelMap.putIfAbsent(siteDefaultLanguageId, objectAction.getName());

        objectAction.setLabel(() -> labelMap);

Reply:
This means no need to add "String defaultLanguageId" to 
```
private void _addObjectDefinitionResources(
```

Because defaultLanguageId comes from the objectDefinition too.

Reply:
But maybe here too we could go through objectAction.getLabel() with for loop as above in the util class, since there can be a lot of locales (if they gave more translations) in the actions labels too.

```
  "objectActions": [
    {
      "label": {
        "en_US": "formula",
        "de_DE": "das formula",
        "es_ES": "formulario",
      },